docker-compose.yml

```diff
@@ -11,12 +11,12 @@ services:
   parity-integration-tests:
     container_name: parity-integration-tests
     image: livepeer/protocol:latest
-    command: npm run test:integration -- --network=parityDev
+    command: npm run test:integration --network=parityDev
     links:
       - parity-dev
   geth-integration-tests:
     container_name: geth-integration-tests
     image: livepeer/protocol:latest
-    command: npm run test:integration -- --network=gethDev
+    command: npm run test:integration --network=gethDev
```

Comment:
removed `--` as it is not required by hardhat.

Reply:
This will simply ignore `--network=gethDev` , this entire flag should be removed. 

AFAIK `npm run` requires `--` to pass through additional arguments or flags that belong to the command that is called underlying by npm

```
npm run <command> [-- <args>]
```

Reply:
ohh! thanks, i wasn't aware of this